src/transformStateWithClones.js

```diff
@@ -65,7 +65,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let cloneState = { ...state };
```

Comment:
```suggestion
  const cloneState = { ...state };
```
you never reassign this variable in your code. Use 'const' instead

Reply:
We can`t use const here, because on line N 74 we reassign cloneState